sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1642,6 +1653,9 @@ def get_transceiver_loopback(self):
         host_input_loopback_lane8   = BOOLEAN                          ; host side input loopback enable lane8
         ========================================================================
         """
+        loopback_support = self.get_transceiver_loopback_support()
```

Comment:
This check is not required if you add the check inside the different loopback APIs, see comments below

Reply:
Need to check if it is flat memory, otherwise won't be able to read page 0x13

Reply:
i meant we can have one single API get_loopback_capability() which will check if its flat memory and also check other loopback capability. we don't need get_transceiver_loopback_support() and get_loopback_capability()